singularity-bootstrap-prefix.def

```diff
@@ -1,16 +1,35 @@
 Bootstrap: docker
-From: centos:8.2.2004
+From: awesomebytes/gentoo_prefix_ci_prepare
+Stage: spython-base
+
+# This container installs Gentoo Prefix noninteractively in 
 
 %post
-    dnf install -y gcc gcc-c++ make diffutils
-    chmod 755 /usr/local/bin/bootstrap-prefix.sh
+# Prepared Ubuntu 16.04
 
-%environment
-    export LC_ALL=C
-    export PATH=/usr/local/bin:$PATH
+# Let's get some specs of the machine that is running this job
+cat /proc/cpuinfo && cat /proc/meminfo && df -h
+
+mkdir /scratch
 
-%files
-    bootstrap-prefix.sh /usr/local/bin
+cd /usr/local/bin
+# Get EESSI or Official bootstrap script
+# wget https://gitweb.gentoo.org/repo/proj/prefix.git/plain/scripts/bootstrap-prefix.sh
+wget https://raw.githubusercontent.com/EESSI/compatibility-layer/master/bootstrap-prefix.sh
```

Comment:
Aha, the old approach assumes that you build the container from the git checkout directory. Pros and Cons there.

Reply:
I expected that most people would clone the repo and build it from there, but maybe that's not true ;-)
I'm okay with doing it like this, but maybe we should make it more clear that it will, by default, always use the latest version from our git repo. This would fit in nicely just before the sentence "If you want to run your own version...".
For instance:

Note: the container will always download and use the latest version of our bootstrap script (taken from the master branch of our Github repository). If you want to run a different version, ...

Reply:
And the indentation and blank lines in the %post section still need to be fixed :-)